modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts

```diff
@@ -121,6 +121,9 @@ export default class Tiles3DConverter {
     await this._addChildren(rootNode, rootTile, 1);
 
     const tileset = transform({root: rootTile}, tilesetTemplate());
+    if (!tileset.root.refine) {
```

Comment:
Set it in the tile initialization https://github.com/visgl/loaders.gl/blob/eec7089c91b2da0a42756aa149469967676c4087/modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts#L113 . There is no a case when it can be "ADD"